packages/app/src/api/zapehr.ts

```diff
@@ -39,7 +44,7 @@ export async function createTelemedRoom(
   }
 }
 
-export async function getSlugAvailability(slug: string): Promise<ZambdaFunctionResponse['response']> {
+export async function getSlugAvailability(slug: string | undefined): Promise<ZambdaFunctionResponse['response']> {
```

Comment:
This one is related to the getSlugAvailability not working properly for now, I added a temporary "undefined" to avoid getting errors locally. I can do a FE handle, but I think Samir's fix will already have conflicts with this one, so I think it would be better if we manage it after either merge.